collectors/tidpost

```diff
@@ -107,8 +125,33 @@ def find_sensor_by_metric(sensor_list, metric):
             return sensor
     raise KeyError('"%s" not found' % metric)
 
-
 def post_sensor_data(report_data, device, prefix=''):
+    now = datetime.datetime.utcnow()
+    sensor_data = {
+        'timestamp': now.isoformat() + "+00:00"
+    }
+
+    for metric, data in report_data['data'].items():
+        server_metric = prefix + metric
+        sensors_coll = device.rels['ch:sensors']
+        try:
+            sensor = find_sensor_by_metric(sensors_coll.rels['items'],
+                server_metric)
+        except KeyError:
+            sensor = sensors_coll.create(
+                {'metric': server_metric, 'unit': data['unit']},
+                auth=COLLECTOR_AUTH)
+        history = sensor.rels['ch:dataHistory']
+
+        sensor_data['value'] = data['value']
+        logger.info("Posting %s data: %s (v2)", server_metric, sensor_data)
+        history.create(sensor_data, cache=False, auth=COLLECTOR_AUTH)
+
+def post_sensor_data_legacy(report_data, device, prefix=''):
```

Comment:
I think the recursive call [here](https://github.com/ResEnv/chain-api/pull/63/files#diff-2ede0ade026f102164e627fcc46b9b1cL143) also get changed to `post_sensor_data_legacy`.